src/plugins/stake-plugins/contracts.js

```diff
@@ -29,62 +29,69 @@ type ContractInfo = {
-  },
-  TOMB_WFTM_LP: {
-    abi: UNISWAP_V2_PAIR,
-    address: '0x2A651563C9d3Af67aE0388a5c8F89b867038089e'
-  },
-  TSHARE_WFTM_LP: {
-    abi: UNISWAP_V2_PAIR,
-    address: '0x4733bc45eF91cF7CcEcaeeDb794727075fB209F2'
-  },
-  TSHARE: {
-    abi: TSHARE_ABI,
-    address: '0x4cdF39285D7Ca8eB3f090fDA0C069ba5F4145B37'
-  },
-  TSHARE_REWARD_POOL: {
-    abi: TSHARE_REWARD_POOL_ABI,
-    address: '0xcc0a87F7e7c693042a9Cc703661F5060c80ACb43'
-  }
+  CEMETERY_V2_REWARD_POOL: { abi: TSHARE_REWARD_POOL_ABI, address: '0x1F832dfBA15346D25438Cf7Ac683b013Ed03E32f' },
+  SPOOKY_SWAP_ROUTER: { abi: UNISWAP_V2_ROUTER_02, address: '0xF491e7B69E4244ad4002BC14e878a34207E38c29' },
+  TOMB_MASONRY: { abi: MASONRY_ABI, address: '0x8764de60236c5843d9faeb1b638fbce962773b67' },
+  TOMB_SWAP_ROUTER: { abi: UNISWAP_V2_ROUTER_02, address: '0x6d0176c5ea1e44b08d3dd001b0784ce42f47a3a7' },
+  TOMB_TREASURY: { abi: TOMB_TREASURY_ABI, address: '0xF50c6dAAAEC271B56FCddFBC38F0b56cA45E6f0d' },
+  TSHARE_REWARD_POOL: { abi: TSHARE_REWARD_POOL_ABI, address: '0xcc0a87F7e7c693042a9Cc703661F5060c80ACb43' },
+
+  // Tokens
+  AVAX: { abi: WRAPPED_FTM_ABI, address: '0x511d35c52a3c244e7b8bd92c0c297755fbd89212' },
+  BNB: { abi: WRAPPED_FTM_ABI, address: '0xd67de0e0a0fd7b15dc8348bb9be742f3c5850454' },
+  BTC: { abi: WRAPPED_FTM_ABI, address: '0x321162cd933e2be498cd2267a90534a804051b11' },
+  CRV: { abi: WRAPPED_FTM_ABI, address: '0x1e4f97b9f9f913c46f1632781732927b9019c68b' },
+  DAI: { abi: WRAPPED_FTM_ABI, address: '0x8d11ec38a3eb5e956b052f67da8bdc9bef8abf3e' },
+  ETH: { abi: WRAPPED_FTM_ABI, address: '0x74b23882a30290451a17c44f4f05243b6b58c76d' },
+  FTM: { abi: WRAPPED_FTM_ABI, address: '0x21be370D5312f44cB42ce377BC9b8a0cEF1A4C83' },
+  FUSD: { abi: WRAPPED_FTM_ABI, address: '0xad84341756bf337f5a0164515b1f6f993d194e1f' },
+  FUSDT: { abi: WRAPPED_FTM_ABI, address: '0x049d68029688eabf473097a2fc38ef61633a3c7a' },
+  LIF3: { abi: WRAPPED_FTM_ABI, address: '0xbf60e7414ef09026733c1e7de72e7393888c64da' },
+  LINK: { abi: WRAPPED_FTM_ABI, address: '0xb3654dc3d10ea7645f8319668e8f54d2574fbdc8' },
+  LSHARE: { abi: WRAPPED_FTM_ABI, address: '0xcbe0ca46399af916784cadf5bcc3aed2052d6c45' },
+  MAI: { abi: ANYSWAP_V5_ERC20_ABI, address: '0xfB98B335551a418cD0737375a2ea0ded62Ea213b' },
+  MIM: { abi: WRAPPED_FTM_ABI, address: '0x82f0b8b456c1a451378467398982d4834b6829c1' },
+  TBOND: { abi: WRAPPED_FTM_ABI, address: '0x24248cd1747348bdc971a5395f4b3cd7fee94ea0' },
+  TOMB: { abi: TOMB_ABI, address: '0x6c021Ae822BEa943b2E66552bDe1D2696a53fbB7' },
+  TREEB: { abi: WRAPPED_FTM_ABI, address: '0xc60d7067dfbc6f2caf30523a064f416a5af52963' },
+  TSHARE: { abi: TSHARE_ABI, address: '0x4cdF39285D7Ca8eB3f090fDA0C069ba5F4145B37' },
+  USDC: { abi: WRAPPED_FTM_ABI, address: '0x04068da6c83afcfa0e13ba15a6696662335d5b75' },
+  WFTM: { abi: WRAPPED_FTM_ABI, address: '0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83' },
+  ZOO: { abi: WRAPPED_FTM_ABI, address: '0x09e145a1d53c0045f41aeef25d8ff982ae74dd56' },
```

Comment:
These shouldn't be wrapped tokens (`WRAPPED_FTM_ABI`). It would make sense to use a `ERC20_ABI` here.